src/assets/i18n/de_DE.json

```diff
@@ -1066,7 +1066,7 @@
   "product.reviews.your_review.heading": "Ihre Produktbewertung",
   "product.short_description.label": "Kurzbeschreibung",
   "product.specific_attributes.label": "Spezifische Attribute",
-  "product.variations.text": "{{ 0, plural, zero{keine Variationen} one{eine Variation} other{# Variationen}}}",
+  "product.variations.text": "{{ 0, plural, zero{Keine Variationen} one{Eine Variation} other{# Variationen}}}",
```

Comment:
@shauke Please check if the capitalization is fine. Having seen your screenshots in [ISSUE 98989](https://dev.azure.com/intershop-com/Products/_workitems/edit/98989), I thought this would be wise.

Reply:
This looks good